sympy/integrals/tests/test_failing_integrals.py

```diff
@@ -229,6 +241,8 @@ def test_issue_9101():
 
 
 @XFAIL
+@slow
+@SKIP("Same as test_W3")
 def test_issue_7161():
     assert not integrate(sqrt(x + 1/x - 2), (x, 0, 1)).has(Integral)
```

Comment:
If this is the same as another test can it not just be deleted?

Reply:
I was considering that as well. My motivation was that W3 is a Wester test (which shouldn't be removed), while this is for a SymPy issue. Although they happens to be the same. On the other hand, in test_W3, the issue number is noted, so it should be OK to remove `test_issue_7161` completely.